skycultures/tikuna/description.en.utf8

```diff
@@ -6,23 +6,23 @@
-<h5 align="center"><img src="Atlas_das_terras_Ticuna.png" width="753" height="453" /><br/>
-<em>Figura 1: The Ticuna Atlas, Figure I<cite id="cit1a"><a href="#cit1"><B>¹</B></a></cite> </em></h5>
+<h5 align="center"><img src="Atlas_das_terras_Tkicuna.png" width="753" height="453" /><br/>
+<em>Figura 1: The Tikuna Atlas, Figure I<cite id="cit1a"><a href="#cit1"><B>¹</B></a></cite> </em></h5>
 <p align="justify">The agencies and researchers that deal with them estimate that more than 53,544 (Siasi/Sesai, 2014) Tikunas live in Brazil (FUNASA, 2009), 8.000 in Colombia (Goulard, 2010), and 6.982 in Peru (INEL, 2007)<cite id="cit2a"><a href="#cit2"><B>²</B></a></cite> . Together, they constitute the Amazonian largest indigenous population. Aside from their own Tikuna language, they also speak Spanish and Portuguese, depending on whether they live in Colombia and Peru or in Brazil.</p>
 <p align="justify">The Tikuna live in riverside villages along the Amazonas-Solimões and its tributaries, called igarapés. Their daily life revolve around the cultivation cycles of different agricultural goods; specially, manioc, used to produce the flour the Tikuna sell in the markets of the region’s main cities. They are also fishers, hunters, and gatherers of the forest products. The women make hammocks, bags, and other objects with plant fibers; and the men paint on tururi fiber and fabricate arrows, paddles, and canoes.</p>
 <p align="justify">The Tikuna live in riverside villages along the Amazonas-Solimões and its tributaries, called igarapés. Their daily life revolve around the cultivation cycles of different agricultural goods; specially, manioc, used to produce the flour the Tikuna sell in the markets of the region’s main cities. They are also fishers, hunters, and gatherers of the forest products. The women make hammocks, bags, and other objects with plant fibers; and the men paint on tururi fiber and fabricate arrows, paddles, and canoes.</p>
 <p align="justify">It should be noted that, in Brazil, the Tikuna Indians live in indigenous lands recognized by Brazil’s indigenous policy. They are located in “terrestrial islands” among non-indigenous territories occupied by the Brazilian Population, from Tabatinga to Manaus (As illustration seen in Brazilian Tikuna Atlas).</p>
 <h3 align="center">The Astronomic Correlations</h3>
 <p align="justify">According to Vieira’s interpretation, the form of Baweta (the Turtle Collective) is associated with the stars witThe “V” shape that Western astronomy relates to the head of Taurus is called Coyatchicüra (Caiman’s Jaw). Wücütcha’s Leg corresponds to the stars that groups as Orion. Its toes are Orion’s Belt and the bend of the legs are ι (iota) Orionis; κ (capa) Orionis; η (eta) Leporis and δ (delta) Leporis. </p>
```

Comment:
“ (capa)” - a typo, it should be (cappa)

Reply:
thank you correction made

Reply:
“witThe” - a typo

Reply:
thank you correction made